src/nativeRenderManager.js

```diff
@@ -48,25 +48,25 @@ export function newNativeRenderManager(win) {
   }
 
   // START OF MAIN CODE
-  let renderNativeAd = function(nativeTag) {
+  let renderNativeAd = function(doc, nativeTag) {
     window.pbNativeData = nativeTag;
     sendMessage = prebidMessenger(nativeTag.pubUrl, win);
     const nativeAssetManager = newNativeAssetManager(window, nativeTag.pubUrl);
 
     if (nativeTag.hasOwnProperty('adId')) {
```

Comment:
When I was running a basic test where I had a GAM creative setup with the new basic template (from the updated README.md) and calling a native creative, the new logic for the `native.js` ran into an issue here b/c the `adId` is not part of the `nativeTag` object that gets passed into the creative's renderAd() function; the adId is still stuck inside the targetingMap param in its natural format.  See below for a copy of the creative code in its initial state when it gets to the code at this point:
```
  var ucTagData = {};
  ucTagData.adServerDomain = "";
  ucTagData.pubUrl = "http://test.localhost:9999/integrationExamples/gpt/hello_world_puc_adformat_nonsf.html?pbjs_debug=true";
  ucTagData.targetingMap = {"hb_adid":["3b4e49b4f268f3"],"hb_adid_appnexus":["3b4e49b4f268f3"],"hb_bidder":["appnexus"],"hb_bidder_appnexus":["appnexus"],"hb_format":["native"],"hb_format_appnexus":["native"],"hb_pb":["5.00"],"hb_pb_appnexus":["5.00"],"hb_size":["0x0"],"hb_size_appnexus":["0x0"]};
  ucTagData.requestAllAssets = true;

  try {
    ucTag.renderAd(document, ucTagData);
  } catch (e) {
    console.log(e);
  }
```  

Because it can't find the `adId`, the rest of the rendering process is aborted.

In the previous way with the `native-render.js` approach, the `adId` a direct part of the object that's passed to the render function, see below for reference:
```
let pbNativeTagData = {};
pbNativeTagData.pubUrl = "https://sneezy.aws.rubiconproject.com/rp-eng-soln/prebid/prebid_1001_native_case4.html?pbjs_debug=true";
pbNativeTagData.adId = "3b8572ff0e96ab";
pbNativeTagData.requestAllAssets = true;
window.pbNativeTag.renderNativeAd(pbNativeTagData);
```

Reply:
Hi @jsnellbaker , the error is in the readme.md file. The right creative should be: 

```
<script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/%%PATTERN:hb_format%%.js"></script>
<script>
  var ucTagData = {};
  ucTagData.adServerDomain = "";
  ucTagData.pubUrl = "%%PATTERN:url%%";
  ucTagData.targetingMap = %%PATTERN:TARGETINGMAP%%;
  ucTagData.hbPb = "%%PATTERN:hb_pb%%";
  ucTagData.hbFormat = "%%PATTERN:hb_format%%";
  ucTagData.adId = "%%PATTERN:hb_adid%%";
  ucTagData.requestAllAssets = true;

  try {
    ucTag.renderAd(document, ucTagData);
  } catch (e) {
    console.log(e);
  }
</script>
```

Going to update the readme file.